eng/pipeline/go-docker-rolling-internal-pipeline.yml

```diff
@@ -22,6 +22,7 @@ extends:
       tsa:
         enabled: true
         configFile: $(Build.SourcesDirectory)/eng/compliance/tsaoptions.json
+      codeql:
```

Comment:
Does this makes any difference?  The 1ES docs for CodeQL says that CodeQL is enabled by default:

> By default, 1ES PT injects CodeQL only for the default branch of the repo.

source: https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/1es-codeql

Reply:
Good question, my dashboard seems to think it's not enabled, I'll do some digging

Reply:
The dashboard says `Database failed to finalize or no source code was built!`, and the build logs for the `CodeQL Initialize` [point at the last-submitted set of requests](https://dev.azure.com/dnceng/internal/_build/results?buildId=2410285&view=logs&j=fc59f0f2-c1bd-58ae-b870-833d1e8a924c&t=ad910901-207c-55f5-a00f-c4453afdfd7e&l=226), which both failed:

pwsh: https://codeql.microsoft.com/job/44362253-19e3-4e33-80ee-96195a66abf5
go: https://codeql.microsoft.com/job/5695f1fb-d910-48d9-8c0e-9da997f8725f

Probably because it didn't detect any source code. As far as I can tell, it isn't easily able to find PowerShell scripts, and we don't run/build any Go programs for it to hook into, we're just using eng/ci-tools to pin some references. (Detection is done by heuristic, and it sees that we do have PowerShell scripts and probably sees eng/ci-tools.)